src/CommandAnalysis.cc

```diff
@@ -639,8 +639,8 @@ void CommandAnalysis::evaluate(const MemorySpecification& memSpec,
 
           sref_ref_act_cycles += sref_duration;
 
-          // FIXME: Here should be ((memSpec.memTimingSpec.RFC - memSpec.memTimingSpec.RP) - sref_duration) to avoid overlapping !!
-          int64_t spup_act = memSpec.memTimingSpec.RFC - sref_duration;
+          int64_t spup_act = (memSpec.memTimingSpec.RFC -
+                              memSpec.memTimingSpec.RP) - sref_duration;
 
```

Comment:
Only this change affects the output (the value assigned to spup_act has changed). It corresponds to the second commit.

The first commit was intended to bring minor improvements to the documentation.